### PR TITLE
Updating readme

### PR DESCRIPTION
**Pull request by ravjain-adb**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ Adobe Experience Platform - Messaging is currently in BETA. Use of this code is
 By using the Beta, you hereby acknowledge that the Beta is provided "as is" without warranty of any kind. Adobe shall have no obligation to maintain, correct, update, change, modify or otherwise support the Beta. You are advised to use caution and not to rely in any way on the correct functioning or performance of such Alpha and/or accompanying materials.
 
 ## About this project
-The AEPMessaging extension allows you to send push notification tokens and push notification click-through feedback to the Adobe Experience Platform.
+The AEPMessaging extension enables sending and tracking push notifications in the Adobe Experience Platform.
```
Comment (PravinPK) [inline README.md:9]: `The AEPMessaging extension, along with Adobe Customer Journey Optimizer, enables sending and tracking push notifications in the Adobe Experience Platform iOS SDK. ` ?

File: README.md [hunk 2]
```diff
@@ -16,23 +16,36 @@ Integrate the Messaging into your app by including the following in your gradle
...
 
-## Run the Demo app
-- Clone or download the project.
-- Open the project in Android Studio.
-- Click the play button from android studio to run the app.
+## Related Projects
+
+| Project                                                      | Description                                                  |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| [AEPEdgeIdentity Extension](https://github.com/adobe/aepsdk-edgeidentity-android) | The AEPEdgeIdentity enables handling of user identity data from a mobile app when using the AEPEdge extension. |
```
Comment (PravinPK) [inline README.md:46]: can we also link edge extension

File: code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MessagingApplication.kt [hunk 3]
```diff
@@ -32,17 +32,20 @@ class MessagingApplication : Application() {
...
         MobileCore.start {
-            // Necessary property id for MessagingSDKTest which has the edge configuration id needed by aep sdk
+            // Necessary property id which has the edge configuration id needed by aep sdk
             MobileCore.configureWithAppID("<appId>")
             MobileCore.lifecycleStart(null);
 
-            MobileCore.updateConfiguration(mutableMapOf("messaging.eventDataset" to "<expEventDatasetId>") as Map<String, Any>?)
+            // To configure the expereience event dataset for push tracking follow this url (https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/)
+            // Uncomment the below line if eventDataset is not configured in the messaging extension in launch
+            //MobileCore.updateConfiguration(mutableMapOf("messaging.eventDataset" to "<expEventDatasetId>") as Map<String, Any>?)
```
Comment (PravinPK) [inline code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MessagingApplication.kt:41]: Is there a common scenario where we need to manually configure `datasetID` manually ?

**Review by PravinPK — APPROVED** [review]

File: code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MessagingApplication.kt (hunk 3 above)
Comment (ravjain-adb) [reply code/app/src/main/java/com/adobe/marketing/mobile/messagingsample/MessagingApplication.kt:41]: Nope

**Pull request merged**